src/idl_gen_swift.cpp

```diff
@@ -162,13 +162,16 @@ class SwiftGenerator : public BaseGenerator {
 
   bool generate() {
     code_.Clear();
+    code_.SetValue("IMPLEMENTATION", parser_.opts.swift_implementation_only
```

Comment:
I think since you only use this in once place, just do a simple:
```
if (parser_.opts.swift_implementation_only) {
  code_ += "@_implementationOnly";
}
```